app.js

```diff
@@ -28,6 +28,13 @@ app.locals.title = `${capitalized(projectName)}- Generated with IronGenerator`;
 const index = require("./routes/index");
 app.use("/", index);
 
+const celebrities = require("./routes/celebrities");
+app.use("/", celebrities);
+
+const movies = require("./routes/movies");
+app.use("/", movies);
```

Comment:
I would use "/movies" as the base route, and then continue based on that in movies.js.
Same goes for Celebrities.